### ISSUE TITLE
Irregular Firefox Downloads

### ISSUE THREAD

**Issue by phume03** [mozilla-mobile/fenix#26143]
Firefox 99.2.0 (Build #2015874211), 
3fbd94a59+
AC: 99.0.10, 2cf4dbe50f
GV: 99.0.1-20220411174855
AS: 91.1.1
on Android mobile sometimes has irregular downloads that supposedly write to /dev/null. Which is a "throw away device" on *nix systems. They do not indicate a file size, nor do they indicate an estimated time of download.

In addition, some downloads terminate irregularly. They do not showu up on the browser's downloads history (as incomplete downloads). Nor, do they show up on the downloads app (or Documents 7.1.1). Please note that this is irregular because if an internet connection fails during a download, it is displayed in the download history with an option to continue (restart/try again) or cancel the download.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21089)

**Comment by cadeyrn** [comment]
> Firefox 99.2.0 

Pleasae test with a **current** version of Firefox. Do you still have problems?

**Comment by LaurentiuApahideanSV** [comment]
@phume03 Could you provide us with some examples of the files you had issues downloading?

**Comment by phume03** [comment]
Hi Laurentiu,

No, I cannot do that. Infact, if I recall clearly it has happened with
different large (above 100MB) and small (less than 50MB) size file formats
(and things in between). I figured it might be an observed phenomenon that
had not yet been raised. But yes, instead of the usual descriptor/name of
the file being downloaded, the initiated download link appears not to start
downloading, and then starts randomly some seconds or a few minutes later.
It does not have the usual progress bar, similar to file downloads where
the server has not provided file size, so the progress cannot be
determined. The first time I caught it late but I did and saved myself data
I suspect — because who knows what was being downloaded and how big it was.
The only description was "downloading /dev/null".

Regards,

Phume

On Monday, July 25, 2022, Laurentiu Apahidean ***@***.***>
wrote:
> @phume03 Could you provide us with some examples of the files you had
issues downloading?
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you were mentioned.Message ID:
***@***.***>

-- 
Yours Sincerely,

Phumelela B. Mdluli
Lake Forest College '15
Waterford Kamhlaba '10 '08
